src/main/resources/staff/templates/document/legalSealedClaim.html

```diff
@@ -298,10 +310,10 @@
     <tr>
       <td><h4>Statement of value</h4></td>
       <td>
-        {% if statementOfValue.personalInjury is defined %}
+        {% if statementOfValue.personalInjury.length > 0 %}
         <span>{{ statementOfValue.personalInjury }} </span> <br/><br/>
         {% endif %}
-        {% if statementOfValue.housingDisrepair is defined %}
+        {% if statementOfValue.housingDisrepair.length > 0 %}
```

Comment:
why length check instead defined??

Reply:
StringBuilder.toString() always creates a new object with empty. and it was adding un necessary line breaks as it was always defined (making condition always true).

Reply:
is not empty